.github/workflows/maven-publish.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       - name: Set up Java for publishing to Maven Central Repository
         uses: actions/setup-java@v1
         with:
-          java-version: 1.8
+          java-version: 11
```

Comment:
Does it make sense to use java 17 ?

Reply:
No java 17 features are required so that LTS version 11 will be used to maximize the compatibility.